tests/network/unit/netinfo_test.py

```diff
@@ -31,7 +31,9 @@
 from vdsm.network.ip.address import prefix2netmask
 from vdsm.network.link import nic
 from vdsm.network.link.bond import Bond, bond_speed
-from vdsm.network.netinfo import addresses, bonding, misc, nics, routes
+from vdsm.network.link.setup import NmstateBridgeOpts
+import vdsm.network.netinfo.nics
```

Comment:
Import of whole module shouldn't be mixed in with partial imports. Also `nics` is there only once it seems, it could be just import of that.

Reply:
done